lib/developer_portal/lib/liquid/drops/site.rb

```diff
@@ -6,7 +6,7 @@ class Site < Drops::Base
 
       drop_example "Using site drop in liquid.", %{
         <div>Organization name {{ site.company_name }}</div>
-        <div>Domain (accessed through account) {{ site.account.domain }}</div>
+        <div>Domain (accessed through account) {{ site.account.external_domain }}</div>
```

Comment:
Related to `lib/developer_portal/lib/liquid/drops/provider.rb`, I don't think this should change.